opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
   // described by http://unitsofmeasure.org/ucum.html.
   string unit = 3;
 
-  // Type of the metric. It describes how the data is reported.
-  //
-  // A gauge is an instantaneous measurement of a value.
-  //
-  // A counter/cumulative measurement is a value accumulated over a time
-  // interval. In a time series, cumulative measurements should have the same
-  // start time, increasing values, until an event resets the cumulative value
-  // to zero and sets a new start time for the subsequent points.
-  enum Type {
-    // Do not use this default value.
+  // Interval describes how metric values are reported over time.
+  enum Interval {
```

Comment:
I don't love this name, since INSTANTANEOUS is kind of "not an interval". Is `TemporalQuality` too verbose?

Reply:
I originally was going to go with `Temporality` or similar `Temporal*` but was worried about the definition being confused with the [a secular possession definition](https://www.merriam-webster.com/dictionary/temporality):

> 1a: civil or political as distinguished from spiritual or ecclesiastical power or authority
> 1b: an ecclesiastical property or revenue —often used in plural

Reply:
I don't like `Interval`, it reminds me that sometimes statisticians distinguish the kind of a number as being an "interval" or "ratio" measurement, a distinction that is close to the debates we're having in OTEP 93.

`Temporality` sounds good to me; I personally will have no confusion over the ecclesiastical usage. `TemporalQuality` is good too. :)

Reply:
I've been trying to use the word "Kind" to indicate something that belongs to a certain category, and "Type" when it refers to a certain value. Maybe "TemporalKind"?

Reply:
I'm onboard abandoning `Interval`. I'm leaning towards `Temporality`, but not entirely opposed to `TemporalQuality`. The `*Quality` part feels a bit redundant (especially if going with the suggestion to make these all "`refinement`" flags) but could be convinced otherwise.

@jkwatson : thoughts?

Reply:
I'm fine with `Temporality`.